src/pkg/transform/git_test.go

```diff
@@ -87,6 +88,7 @@ func TestGitTransformURLSplitRef(t *testing.T) {
 		{"https://github.com/defenseunicorns/zarf.helm.git", ""},
 		{"https://github.com/defenseunicorns/zarf.git", "refs/tags/v0.16.0"},
 		{"https://github.com/DoD-Platform-One/big-bang.git", "refs/heads/release-1.54.x"},
+		{"https://github.com/prometheus-community/helm-charts.git", "kube-prometheus-stack-47.3.0"},
```

Comment:
The two test cases we would need to cover were:

Trailing slash after a path
```
https://github.com/prometheus-community/
```

Trailing slash with no path
```
https://github.com/
```

Reply:
misinterpreted this somehow, now should be addressed. lmk